src/Lykke.MatchingEngine.Connector/Helpers/MeConnectorContainerBuilderExtensions.cs

```diff
@@ -27,14 +27,14 @@ public static class MeConnectorContainerBuilderExtensions
         }
 
         public static void BindMeClient(this ContainerBuilder ioc,
-            IPEndPoint ipEndPoint, ISocketLog socketLog = null)
+            IPEndPoint ipEndPoint, ISocketLog socketLog = null, bool ignoreErrors = false)
         {
             if (socketLog == null)
                 socketLog = new SocketLogDynamic(i => { },
                     str => Console.WriteLine(DateTime.UtcNow.ToIsoDateTime() + ": " + str)
```

Comment:
Is it correct to use Console? Why not ILog?

Reply:
I don't like using of Console.WriteLine at all, but have decided to not change this code for compatibility reasons. It a code bellow a changed Console.WriteLine into ISocketLog.Add.